NavigationReactNative/src/cpp/react/renderer/components/navigationreactnative/NVBarButtonShadowNode.cpp

```diff
@@ -1,4 +1,5 @@
 #include "NVBarButtonShadowNode.h"
+#include "NVSystemImageValidator.h"
```

Comment:
I tried running on Android but it wasn't happy because `NVSystemImageValidator.h` doesn't exist (same for `NVTabBarItemShadowNode`). You can fix it by adding the following `ifdef`'s. I don't know if there's an IOS identifier so I used ANDROID with else directives instead. Do you know if there's a better way?
```c++
#ifdef ANDROID
#else
#include "NVSystemImageValidator.h"
#endif
...
#ifdef ANDROID
  bool isSystemImageResult = false;
#else
  bool isSystemImageResult = isSystemImage(newImageSource.uri);
#endif
```

Reply:
I don't think there is much to do, but we can use bang operator for truthy/falsy values afaik? Maybe that would reduce the first part.

```suggestion
#ifdef !ANDROID
#include "NVSystemImageValidator.h"
#endif
```